Corrects the Escape Alone text

Dorsisdwarf opened the pull request:
A while ago, Escape Alone was updated so that you could bring as many antagonists on the shuttle with you as you liked, but the description doesn't hint that this is the case. This PR fixes that

Dorsisdwarf committed:
Fixes the description of the escape alone text to make it clearer you can take your traitorbuddy

Dorsisdwarf committed:
Updates for fucking wizards

Dorsisdwarf committed:
words

Jalleo commented:
Silicons and pai are not counted i believe

Incoming5643 commented:
Silicons don't count because they're registered as dead, I'm not 100% sure about pAIs considering how in flux the pAI mob has been in over the ages

Incoming5643 commented on code/game/gamemodes/objective.dm:
```diff
@@ -186,7 +186,7 @@
 
 /datum/objective/hijack
-	explanation_text = "Hijack the emergency shuttle by escaping alone."
+	explanation_text = "Hijack the emergency shuttle by ensuring that only non-Nanotrasen entities escape alive, aboard, and not in the shuttle brig."
```
I really wish there was a more terse way of phrasing this, as I'm pretty sure that's why I didn't end up changing it initially.

phil235 replied:
Agreed, the text is a bit too long and awkward.

ojacobson replied:
> Hijack the emergency shuttle by making sure no Nanotrasen employees ride outside the brig.

Iamgoofball commented:
why not change it to

"Hijack the shuttle and ensure that only antagonists escape on board the shuttle."

Jalleo commented:
If non antags are in the shuttle brig they arent counted btw
How about:
Hijack the shuttle; ensure all real Nanotrasen workers arent free by arrival.

bustygoku commented:
Hijack the emergency shuttle by escaping alone or with other syndicate agents

Dorsisdwarf commented:
Bustygoku, the problem with that is that it's not just syndies.

Dorsisdwarf committed:
We spent twenty minutes arguing about this

Updates desc to "Hijack the shuttle to ensure no loyalist Nanotrasen crew escape alive and in custody"

Jalleo commented:
Best it can get really

Jordie0608 commented on code/game/gamemodes/objective.dm:
```diff
@@ -186,7 +186,7 @@
 
 /datum/objective/hijack
-	explanation_text = "Hijack the emergency shuttle by escaping alone."
+	explanation_text = "Hijack the shuttle to ensure no loyalist Nanotrasen crew escape alive and out of custody.
```
Missing a closing "

Dorsisdwarf committed:
Adds a quote to hijack

bustygoku commented:
We should update the lore to specify that the changelings are syndicate since they a) can get traitor objectives/gear and b) usually go on arbitrary syndicate assassination and steal missions in addition to harvesting genomes

Jalleo commented:
Not required due to how its currently worded
